src/components/ImageUploader/src/ImagePicker.vue

```diff
@@ -187,15 +187,15 @@ export default {
 	box-sizing: border-box;
 	width: 96px;
 	height: 96px;
-	background-color: var(--maker-color-background, #fff);
-	border: 1px solid var(--maker-color-neutral-20, rgba(0, 0, 0, 0.15));
-	border-radius: var(--maker-shape-default-border-radius, 8px);
+	background-color: $maker-color-background;
+	border: 1px solid $maker-color-neutral-20;
+	border-radius: $maker-shape-default-border-radius;
```

Comment:
Also need a `fill: $maker-color-neutral-20` somewhere in `.ImagePickerInputContainer` to correctly apply theme colors for the drag icon and input icon.